code-examples/core_features/did/10_did.ts

```diff
@@ -0,0 +1,29 @@
+import { DemoKeystore, DidDetails, DidUtils } from '@kiltprotocol/did'
+
```

Comment:
Is this empty line intentional?

Reply:
It was not 😅 https://github.com/KILTprotocol/docs/pull/101/commits/8d75a54e22badfdd944fc5bff2584b00b2fe4638